initialize/initialize.go

```diff
@@ -128,6 +129,13 @@ func githubSearch(query string) ([]Repo, error) {
 
 	client := github.NewClient(tc)
 
+	if os.Getenv("GITHUB_URL") != "" {
+		baseEndpoint, _ := url.Parse(os.Getenv("GITHUB_URL"))
```

Comment:
we should return + handle these errors

Reply:
I basically cribbed the code direct from https://github.com/google/go-github/blob/master/github/github.go#L310

Perhaps we should be using `NewEnterpriseClient` if `GITHUB_URL` is defined and let it do the error handling?

Reply:
I like the idea of using `NewEnterpriseClient` 👍